pkg/storage/s3/s3.go

```diff
@@ -24,22 +25,22 @@ type S3 struct {
 }
 
 type driver struct {
-	Config     *S3
-	Context    context.Context
-	kubeClient client.Client
+	storageBase.Driver
+	Config *S3
 }
 
 // NewDriver creates a new s3 storage driver
 // Used during bootstrapping
 func NewDriver(ctx context.Context, cfg *configv1.InfrastructureStatus, clnt client.Client) *driver {
-	return &driver{
-		Context: ctx,
+	drv := driver{
 		Config: &S3{
 			Region:    cfg.PlatformStatus.AWS.Region,
 			InfraName: cfg.InfrastructureName,
 		},
-		kubeClient: clnt,
 	}
+	drv.Context = ctx
+	drv.KubeClient = clnt
```

Comment:
Is there a reason you had to split these out?

Reply:
Yeah, it's a Golang quirk.  Otherwise you get this:
```
# github.com/openshift/managed-velero-operator/pkg/storage/s3
pkg/storage/s3/s3.go:51:3: cannot use promoted field Driver.Context in struct literal of type driver
pkg/storage/s3/s3.go:52:3: cannot use promoted field Driver.KubeClient in struct literal of type driver
```